### PR TITLE
add google ads tracking to artwork pages

### PR DESCRIPTION
**Pull request by cavvia**
As with our FB tracking code, this sends a product ID over to Google Ads when users are visiting artwork pages. This is a requirement for retargeting Buy Now works over the Google Ads display network. This is a high priority request from marketing as we've been banned from Criteo and need to use Google as a marketing channel for Buy Now. It takes 24-48 hours for this to start populating pageviews on the Google Ads side.

No additional user data is sent in the request.

I've added the `GOOGLE_ADWORDS_ID` ENV var to staging and production.

### CODE REVIEW

File: src/mobile/analytics/before_ready.js [hunk 1]
```diff
@@ -19,8 +19,6 @@ if (pageType == "artwork") {
   properties["price_listed"] = sd.ARTWORK.price && sd.ARTWORK.price.length > 0
 }
 
-console.log(properties)
```
Comment (mzikherman) [inline src/mobile/analytics/before_ready.js]: lol 🌴

File: src/desktop/apps/artwork/templates/index.jade [hunk 2]
```diff
@@ -6,6 +6,7 @@ append locals
 block head
   include ../components/meta/index
   include ../components/facebook/index
+  include ../components/google/index
```
Comment (mzikherman) [inline src/desktop/apps/artwork/templates/index.jade]: 👍

**Comment by mzikherman** [comment]
> as we've been banned from Criteo

On the bright side I guess that means we can remove the Criteo reports.

**Comment by cavvia** [comment]
> > as we've been banned from Criteo
> 
> On the bright side I guess that means we can remove the Criteo reports.

We are trying our best to unban ourselves unfortunately.

**Pull request merged**